helm_chart/ibm_storage_enabler_for_containers/templates/ubiquity-db-service.yml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: ubiquity-db
+  labels:
```

Comment:
There are some standard labels that recommended by helm, we'd better add them for every resource, see https://docs.helm.sh/chart_best_practices/#standard-labels

Reply:
sure
please add relevant labels in later PRs as needed.
I believe you should add release + chart labels.